Mathlib/Analysis/SpecialFunctions/ContinuousFunctionalCalculus/Rpow/Basic.lean

```diff
@@ -671,6 +671,61 @@ omit [T2Space A] [IsTopologicalRing A] in
 lemma _root_.IsStrictlyPositive.rpow {a : A} {y : ℝ} (ha : IsStrictlyPositive a) :
     IsStrictlyPositive (a ^ y) := by grind
 
+/-- For an element `a` in a C⋆-algebra, TFAE:
+* `a` is strictly positive,
```

Comment:
Could you use numbers in the docstring instead of bullet points? This way potential users don't have to start counting the bullet points to use the theorem.